src/NUnitFramework/framework/Is.cs

```diff
@@ -486,5 +486,18 @@ public static RangeConstraint InRange(object from, object to)
 
         #endregion
 
+        #region Any Of
+
+        /// <summary>
+        /// Returns a constraint that tests if an item is equal to any of parameters
+        /// </summary>
+        /// <param name="expected">Expected values</param>
+        public static AnyOfConstraint AnyOf(params object[] expected)
+        {
+            return new AnyOfConstraint(expected);
```

Comment:
⚠ When people use `Is.AnyOf(null)`, we should use `new object[] { null }` like we do for all our other params APIs.
This doesn't apply to the `AnyOfConstraint(IEnumerable)` constructor itself.

Can we have tests for `Is.AnyOf(null)` and `Builder.AnyOf(null)` in AnyOfTests, please?

Reply:
`Builder.AnyOf` isn't a real thing. It's just a construct used in our syntax tests.

Reply:
Proper handling of null passed to a method taking a params array as it's argument is very difficult. We need to specify exactly what should happen in a test, as you say.

Reply:
@CharliePoole `Builder.AnyOf` is how you'd write the test for `ConstraintExpression.AnyOf` which needs separate handling from `Is.AnyOf`.

Reply:
Right, in the common base syntax tests. You'd just need to add a case that has null as an accepted value and specify what the ToString() should look like. We should do that for sure.

Reply:
Added null value handling, and tests to Syntax tests (AnyOfTests -> AnyOf_NullValue_Tests).

btw, is that common approach to define new tests by inheriting from base class with defined tests, and to change only setUp method? Seems like I hate that, haha :))))

Reply:
@Dreamescaper I've seen variations on that, yes. What do you think could be improved?